### ISSUE TITLE
[Accessibility] Refunds Quick Audit

### ISSUE THREAD

**Issue by shiki** [woocommerce/woocommerce-android#1941]
## Is your feature request related to a problem? Please describe.

This is just a quick TalkBack audit for the Refunds feature.

<img src="https://user-images.githubusercontent.com/198826/74458865-92c12f00-4e47-11ea-984f-cf9255eaa6e8.png" width="240">

### Product Labels

<img src="https://user-images.githubusercontent.com/198826/74458915-a371a500-4e47-11ea-9023-e7e849984bdf.png" width="320">

When focusing on a product, the “Product Image” is mentioned. I believe we can remove that. 🙂 

There is also no mention of the quantity that is being refunded. If I change the quantity to “1”, it is not mentioned when I focused on the product.

### Quantity

<img src="https://user-images.githubusercontent.com/198826/74459377-6e198700-4e48-11ea-94f4-f19de43c825c.png" width="320">

The quantity is separately accessible. It's a bit difficult to discover since it's very small. It might be better to make double-tapping (TalkBack) also activate and show the Quantity Dialog. 

### Quantity Dialog

<img src="https://user-images.githubusercontent.com/198826/74459477-9d2ff880-4e48-11ea-8332-9f031a691930.png" width="320">

The currently selected value is not read as “selected” (or something similar) by TalkBack. It is weirdly read as “Edit Box”. 

### Labels and Values

Current | Better
--------|-------
![image](https://user-images.githubusercontent.com/198826/74459752-129bc900-4e49-11ea-9270-5f668b79617b.png)        |       ![image](https://user-images.githubusercontent.com/198826/74459800-234c3f00-4e49-11ea-90fa-460fb87dcfd8.png)

The labels like “Subtotal” are separately accessible from their values. This makes it easy for the user to lose **context**. It's better if the label and value are grouped as one accessibility element.

--- 

@Garance91540 @0nko

**Comment by Garance91540** [comment]
Thank you for looking into this @shiki !!! Don't hesitate to take a look at our existing features as well :)

**Comment by JorgeMucientes** [comment]
Closing this as part of a maintenance pass on the GitHub issue backlog. This issue matched the auto-close criteria for this pass: it has not been updated since 2020-02-14, is unassigned, is not labeled as a bug or high/medium priority, and does not appear to involve POS, payments, login, data loss, crash, security, vulnerability, auth, or fraud.

If this is still relevant, please comment with current details and we can reopen it or move it into Linear for triage.
